ssl/ech/ech_local.h

```diff
@@ -115,7 +115,7 @@ typedef struct ossl_ctx_ech_st {
     unsigned char *alpn_outer;
     size_t alpn_outer_len;
     SSL_ech_cb_func cb; /* callback function for when ECH "done" */
-} OSSL_CTX_ECH;
+} OSSL_ECH_CTX;
```

Comment:
not that it matters much but please also change `struct ossl_ctx_ech_st` to `struct ossl_ech_ctx_st` for consistency

Reply:
ah, right - also done.